packages/dashboard-frontend/src/pages/WorkspaceDetails/OverviewTab/StorageType/index.tsx

```diff
@@ -230,8 +230,8 @@ export class StorageTypeFormGroup extends React.PureComponent<Props, State> {
           label="Per-user"
           name="per-user"
           id="per-user-type-radio"
-          description="Per-workspace Storage. A PVC is provisioned for each workspace within the namespace.
-        All of the workspace's storage are mounted on subpaths within the workspace's PVC."
+          description="Per-user Storage. One PVC is provisioned per namespace.
+          All of the workspace's storage (volume mounts) mounted in it on subpaths according to devworkspace ID."
```

Comment:
```suggestion
```
I would probably remove this line two


Another option would be changing it to